From d3b7b3b737cf95c344bd4b1e72afc5543548efc8 Mon Sep 17 00:00:00 2001
From: rvuyyurx <Raja.NareshX.Vuyyuru@intel.com>
Date: Tue, 6 Oct 2020 16:44:40 +0530
Subject: [PATCH] [SELINUX] Fix neverallow vendor violators

ATS test failure due to postinstall violating selinux neverallow rule.
Removed neverallow voilator rules.

Signed-off-by: rvuyyurx <Raja.NareshX.Vuyyuru@intel.com>

diff --git a/abota/generic/no_vendor_prefix/postinstall.te b/abota/generic/no_vendor_prefix/postinstall.te
index 2840f55..e26cac9 100644
--- a/abota/generic/no_vendor_prefix/postinstall.te
+++ b/abota/generic/no_vendor_prefix/postinstall.te
@@ -1,4 +1,3 @@
 typeattribute postinstall system_writes_vendor_properties_violators;
-typeattribute postinstall system_executes_vendor_violators;
 set_prop(postinstall, ota_prop)

diff --git a/abota/generic/vendor_prefix/postinstall.te b/abota/generic/vendor_prefix/postinstall.te
index 12eb5e8..333dad5 100644
--- a/abota/generic/vendor_prefix/postinstall.te
+++ b/abota/generic/vendor_prefix/postinstall.te
@@ -1,3 +1,2 @@
 typeattribute postinstall system_writes_vendor_properties_violators;
-typeattribute postinstall system_executes_vendor_violators;
 set_prop(postinstall, vendor_ota_prop)

2.28.0

